.sanity-ansible-ignore-2.9.txt

```diff
@@ -1,24 +1,23 @@
-plugins/module_utils/certificate_lsr/providers/__init__.py empty-init!skip
-plugins/module_utils/certificate_lsr/providers/base.py compile-2.6!skip
-plugins/module_utils/certificate_lsr/providers/base.py import-2.6!skip
```

Comment:
you'll need to keep the `compile-2.6!skip` and `import-2.6!skip` in the ignore files